packages/crew_profile_base.rb

```diff
@@ -0,0 +1,69 @@
+require 'package'
+
+class Crew_profile_base < Package
+  description 'Crew-profile-base sets up Chromebrew\'s environment capabilities.'
+  homepage 'https://github.com/chromebrew/crew-profile-base'
+  version '0.0.1-pre6'
+  license 'GPL-3+'
+  compatibility 'all'
+  source_url 'https://github.com/chromebrew/crew-profile-base/archive/refs/tags/0.0.1-pre6.tar.gz'
+  source_sha256 '0a4ebe740de0d988ef594e8e3dab83dc201e79b0a7318e62a25ae8fd6f1b1575'
+
+  # This package shouldn't have binaries.
```

Comment:
@uberhacker Is there a way to prevent the generation of binaries for this package?
If binaries get generated by this package, installation of said binary will cause the preinstall section of this package file to get ignored during an install, overwriting local changes, I think.

Reply:
Well, just don't create binaries. :)  I have been excluding certain packages in `tools/needs_binaries.sh` if that helps any.